src/index.html

```diff
@@ -11,7 +11,179 @@
+              type="date"
+              id="date"
+              name="birth-date"
+              placeholder="dd.mm.yyyy"
+              min="1900-01-01"
+              max="2023-04-30"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="accept-term">
+            I accept the term of the agreement
+            <input type="checkbox" id="accept-term">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="field">
+          <label for="user-email">
+            E-mail:
+            <input
+              type="email"
+              id="user-email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="user-password">
+            Password:
+            <input
+              type="password"
+              id="user-password"
+              minlength="6"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="field">
+          Do you love cats?
+          <input
+            type="radio"
+            id="yes"
+            name="love-cats"
+            value="yes"
+          >
+          <label for="yes">Yes</label>
+          <input type="radio"
+                 id="no"
+                 name="love-cats"
+                 value="No"
+          >
+          <label for="no">No</label>
+        </div>
+
+        <div class="field">
+          <label for="color">
+            What is your favorite color?
+            <input type="color" id="color">
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="time">
+            What time do you go to bed?
+            <input
+              type="time"
+              id="time"
+              step="1"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="carBrand">
+            What are your favorite brand of cars?
+            <select
+              name="carBrand"
+              id="carBrand"
+              required
+              multiple
+            >
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+              <option disabled></option>
+            </select>
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="workRate">
+            How do you rate our work?
+            <input
+              type="range"
+              id="workRate"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label class="field" for="comments">
+          Comments:
+          <textarea id="comments"></textarea>
+        </label>
```

Comment:
Add a `name` attribute to the textarea element so it can be submitted with the form. For example: `name="comments"`